marketplace/manifest.yaml

```diff
@@ -1,11 +1,11 @@
-applicationName: com.spinkube.spinkube
-publisher: "Spinkube"
+applicationName: dev.spinkube.spinkube
+publisher: "Fermyon"
 description: "SpinKube on Azure Marketplace"
 version: 1.0.0 #Must be in the format of #.#.#
 helmChart: "./charts/spinkube-azure-marketplace"
 clusterArmTemplate: "./mainTemplate.json"
 uiDefinition: "./createUIDefinition.json"
-registryServer: "TODO.azurecr.io"
+registryServer: "spinkubemarketplace.azurecr.io"
 testParameterFile: "./parameterFile.json"
 extensionRegistrationParameters:
   defaultScope: "cluster" # cluster or namespace 
```

Comment:
**question:** should we add `namespace` to be `"spinkube"`? ([ref](https://github.com/Azure-Samples/kubernetes-offer-samples/blob/main/samples/k8s-offer-azure-vote/manifest.yaml#L11))

Reply:
Yes, added!  I somehow removed that line in the last commit; added back.